docs/events/api/timeline-api-import.asciidoc

```diff
@@ -2,6 +2,7 @@
 === Import timelines and timeline templates
 
 Imports timelines and timeline templates from an `ndjson` file.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment:
This one looks a bit off too:
![image](https://user-images.githubusercontent.com/616158/118679860-1be5b500-b7b3-11eb-955e-f10184c4af83.png)